schemas/mod-data-import/dataImportQueueItem.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "File Extension Schema",
+    "additionalProperties": false,
+    "type": "object",
+    "properties": {
+      "id": {
+        "description": "Unique identifier",
+        "$ref": "../../raml-util/schemas/uuid.schema"
+      },
+      "jobExecutionId": {
+        "description": "job execution Id",
+        "$ref": "../../raml-util/schemas/uuid.schema"
+      },
+      "uploadDefinitionId": {
+        "description": "upload Definition Id",
+        "$ref": "../../raml-util/schemas/uuid.schema"
+      },
+      "size": {
+        "description": "Size of the file",
+        "type": "integer"
```

Comment:
Is this the size of the file (in bytes or records)?

Reply:
I dont have a preference here.  We should talk about it but I am going to merge this PR